mods/cnc/fluent/rules.ftl

```diff
@@ -728,17 +729,17 @@ actor-msam =
     .name = Rocket Launcher
     .description =
     Long-range rocket artillery.
-      Strong vs all ground units.
+      Strong vs All Ground Units.
```

Comment:
capitalisation seems all over the place. In some places ground units are capitalised, and not in others